controllers/api_v1/keyword_test.go

```diff
@@ -127,6 +127,183 @@ func TestKeywordAPIControllerDbTestSuite(t *testing.T) {
 	suite.Run(t, new(KeywordAPIControllerDbTestSuite))
 }
 
+func (s *KeywordAPIControllerDbTestSuite) TestFetchKeywordsWithValidParams() {
+	keyword := models.Keyword{UserID: s.user.ID, Keyword: faker.Name()}
+	db.GetDB().Create(&keyword)
+
+	headers := http.Header{}
+	headers.Set("Authorization", "Bearer test-access")
+
+	resp := testHttp.PerformRequest(s.engine, "GET", "/api/v1/keywords", headers, nil)
+	respBodyData, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		log.Error(err)
+	}
+
+	var parsedRespBody map[string][]api_helper.DataResponseObject
+	err = json.Unmarshal(respBodyData, &parsedRespBody)
+	if err != nil {
+		log.Error(err)
+	}
+
+	assert.Equal(s.T(), http.StatusOK, resp.Code)
+	assert.Equal(s.T(), 1, len(parsedRespBody["data"]))
```

Comment:
@gutakk Is it possible to check the keyword name here?

Reply:
Yes it is possible. Checked in 820433731b7e181eaa092e44d72fc92e9bea2cc1